pep-0517.txt

```diff
@@ -164,74 +164,15 @@ with tools that do not use this spec.
 
 The build backend object is expected to have attributes which provide
 some or all of the following hooks. The common ``config_settings``
-argument is described after the individual hooks::
+argument is described after the individual hooks.
 
-  def get_build_wheel_requires(config_settings):
-      ...
-
-This hook MUST return an additional list of strings containing PEP 508
-dependency specifications, above and beyond those specified in the
-``pyproject.toml`` file, to be installed when building a wheel. Example::
-
-  def get_build_wheel_requires(config_settings):
-      return ["wheel >= 0.25", "setuptools"]
-
-Optional. If not defined, the default implementation is equivalent to
-``return []``.
+Mandatory hooks
+===============
 
```

Comment:
To generate a useful API overview in the table of contents, it would be helpful to have subheadings for each hook:

    build_wheel
    -----------

    [signature and specification]

    build_sdist
    -----------

    [signature and specification]

etc